lib/capybara_accessibility_audit/audit_system_test_extensions.rb

```diff
@@ -104,6 +104,8 @@ def assert_no_accessibility_violations(**options)
       axe_matcher = Axe::Matchers::BeAxeClean.new
       axe_matcher = options.inject(axe_matcher) { |matcher, option| matcher.public_send(*option) }
 
+      yield if block_given?
```

Comment:
I don't think this is quite doing what I want. I think I need to ensure it fails with the proper message. Something like: "Expected page to have no accessibility violations but found 1"